src/System.Management.Automation/namespaces/FileSystemProvider.cs

```diff
@@ -7966,26 +7966,6 @@ private struct REPARSE_GUID_DATA_BUFFER
             IntPtr OutBuffer, int nOutBufferSize,
             out int pBytesReturned, IntPtr lpOverlapped);
 
-#if !CORECLR
-
-        [DllImport("kernel32.dll", SetLastError = true, CharSet = CharSet.Unicode)]
-        private static extern IntPtr FindFirstFileName(
```

Comment:
Currently it is not used. The API is specific for Windows file systems. It works with hard links - it is already in PowerShell Core.